hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/ha/TestObserverReadProxyProvider.java

```diff
@@ -434,6 +438,41 @@ public void testExecutionExceptionGetHAServiceStateWithTimeout() throws Exceptio
     verify(logger).debug(eq("Exception in NN probe task for {}: {}"), eq(null), eq(e));
   }
 
+  /**
+   * Test default (no timeout) getHAServiceState when we have a slow NN.
```

Comment:
`no timeout` should be `long timeout`.
I got confused and thought you'd now made the default having no timeout.

Reply:
thanks Simba! my fault.